Vault-Deployment/manifests/network-policies/cilium-policies.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@norbertgruszka knock yourselves out here, I did not find time yet to test the Cilium policies and only tested the Kubernetes policies for now. Maybe in another PR later.